# Service to get inverse kinematics of the robot
float64 x
float64 y
float64 z
float64 q0
float64 qx
float64 qy
float64 qz
---
float64[] joints
bool success
string msg
